package.json

```diff
@@ -65,7 +65,6 @@
   "jspm": {
     "files": [
       "package.json",
-      "bower.json",
```

Comment:
Um, not sure why this line is still in...

Reply:
The diff is showing this line being removed? Unless I'm missing something? (I've really not used Github much so that's entirely possible!)

![image](https://user-images.githubusercontent.com/13396705/43903726-f54fb34c-9be4-11e8-8ae0-c979a959f98e.png)

The diff for the entire branch doesn't show this line being present either:

![image](https://user-images.githubusercontent.com/13396705/43903684-e142e162-9be4-11e8-93cd-4da92dfa3c83.png)

The line isn't present in my working copy and there is nothing to commit / push.

![image](https://user-images.githubusercontent.com/13396705/43903904-5a4913ec-9be5-11e8-9717-2fe45839d695.png)

Reply:
Ah, I see what you mean.

Reply:
This is what I did to pull in your change and apply it to my branch. Am I holding it wrong?

```
git fetch upstream
git checkout dev
git merge upstream/dev

git checkout jspm-package-size
git rebase dev
```